longhorn-cmd

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+cat << EOF
+USAGE:
+  # Need to use quotation marks on [longhorn-command-args] as the 2nd arg of this script
```

Comment:
`-` should work 

Also, must have longhorn installed in `longhorn-system` namespace and have access to `kubectl` and the namespace

Reply:
User shouldn't need to add any extra characters for this to work.